File: quartzy/src/main/java/com/haier/domain/TaskInfo.java
package com.haier.domain;

import lombok.Data;
import lombok.NoArgsConstructor;

import java.io.Serializable;

/**
 * @author ycm
 * @date 2018/1/29
 */
@Data
@NoArgsConstructor
public class TaskInfo implements Serializable {

    private int id = 0;
    /**任务名称*/
    private String jobName;
    /**任务分组*/
    private String jobGroup;
    /**任务描述*/
    private String jobDescription;
    /**任务状态*/
    private String jobStatus;
    /**任务表达式*/
    private String cronExpression;
    /**任务创建时间*/
    private String createTime;
}

File: quartzy/src/main/resources/static/js/task-info.js

var TaskInfo = {

    grid: '#taskinfo-grid',
    form: '#taskinfoForm',

    init: function () {
        this.initGrid();
    },
    // 初始化表格
    initGrid: function () {
        $(this.grid).datagrid({
            toolbar: '#taskinfo-tbar',
            url: '/list',
            method: 'post',
            fitColumns: true,
            striped: true,
            fit: true,
            pagination: false,
            rownumbers: true,
            columns:[[
                {field:'id',title:'',checkbox:'true', width:20},
                {field:'jobName',title:'JobName',width:120},
                {field:'jobGroup',title:'JobGroup',width:50},
                {field:'jobDescription',title:'JobDescription',width:120},
                {field:'jobStatus',title:'JobStatus',width:50},
                {field:'cronExpression',title:'CronExpression',width:60},
                {field:'createTime',title:'CreateTime',width:70}
            ]]
        })
    }

}